### ISSUE TITLE
Enhancement : Sort debug logs descending to list newest log at top

### ISSUE THREAD

**Issue by harshal-gajare** [joeferraro/MavensMate#768]
* **MavensMate Desktop Version**: 0.0.11-beta5

* **Editor**: Any

* **MavensMate Plugin Version**: 7.0.1

* **Platform/Version**: Any

* **Proxy Details**: Any

* **Salesforce API Version**: ANY

* **Salesforce Server**: Any 
 

It would be good to have the debug logs sorted descending to get the latest log at the top. 

![image](https://cloud.githubusercontent.com/assets/5017173/18840441/9b91f008-83dc-11e6-90d8-cf1fa420694c.png)

**Comment by nerdybeast** [comment]
This should probably be up to each individual user.  Most people are going to be used to the debug logs sorting the way they currently do.  A simple "sort" button/icon would allow each user to sort the debug logs as they see fit.

**Comment by ChuckJonas** [comment]
I agree with @nerdybeast that there should be an option to set the sort direction, but I also think the default direction should be descending.  This is more consistent with how other systems (developer console) work and also consistent with how new logs are added to the list.

<img width="653" alt="mavensmate___realself___debug_logs" src="https://cloud.githubusercontent.com/assets/5217568/18925505/72f53b14-8571-11e6-9ab2-2aefa5d0c380.png">
_notice how they are in asc order except for the new log which is added to the top_

I'll submit a PR to set the default to DESC, but we'll let Joe decide if it's the correct behavior.
